sr_robot_launch/test/test_sr_ur_unlock.test

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <!--param name="ra_sr_ur_robot_hw/headless_mode" value="true"/>
+  <param name="la_sr_ur_robot_hw/headless_mode" value="true"/-->
+
+  <test test-name="test_sr_ur_unlock_left" pkg="sr_robot_launch" type="test_sr_ur_unlock_left.py" time-limit="700.0"/>
+  <test test-name="test_sr_ur_unlock_right" pkg="sr_robot_launch" type="test_sr_ur_unlock_right.py" time-limit="700.0"/>
+  <test test-name="test_sr_ur_unlock_bimanual" pkg="sr_robot_launch" type="test_sr_ur_unlock_bimanual.py" time-limit="700.0"/>
```

Comment:
http://docs.ros.org/en/jade/api/rostest/html/rostest-module.html#run might be worth trying to pass the args instead of rewriting 3 times the same code, up to you

Reply:
there is certainly unneeded redundancy here, but those args go into the launch files, and bringing them out into the class wasn't as easy as I'd hoped